core/src/stores/postgres.rs

```diff
@@ -1624,6 +1625,12 @@ impl Store for PostgresStore {
         where_clauses.extend(filter_clauses);
         params.extend(filter_params);
 
+        let (view_filter_clauses, view_filter_params, p_idx) =
+            Self::where_clauses_and_params_for_filter(view_filter, p_idx);
```

Comment:
Why here using `(view_filter, p_idx)` and not `(filter, params.len() + 1)` as for the call for the regular filters?

Reply:
They're kind of equivalent, p_idx is returned by the previous call. params.len() might be slightly more resilient as it will resit adding params in between the calls, but adding a param should increment p_idx to be correct so....